packages/cli/package.json

```diff
@@ -58,6 +58,7 @@
     "eslint": "8.4.1",
     "fancy-test": "1.4.10",
     "jest": "27.4.3",
+    "jest-mock": "^27.4.2",
```

Comment:
you don't need to add  `jest-mock`  dependency here to use only `mocked` . definitions of `mocked` are there in `@types/jest` , see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57776 
 it just my opinion :)

Reply:
Good catch! Thanks!

Reply:
Thanks! I updated to `@types/jest@27.4` and started using `jest.mocked`